ledger-core/conveyor/smachine/slot_machine.go

```diff
@@ -1100,12 +1100,39 @@ func (m *SlotMachine) activateDependantByDetachable(links []StepLink, ignore Slo
 		return true
 	}
 
-	m.syncQueue.AddAsyncUpdate(SlotLink{}, func(_ SlotLink, worker FixedSlotWorker) {
+	if !m.syncQueue.AddAsyncUpdate(SlotLink{}, func(_ SlotLink, worker FixedSlotWorker) {
```

Comment:
I don't insist, but I would like the following code instead of this:
`
added := m.syncQueue.AddAsyncUpdate(...)
if !added {
    ...
}
`
It's just way easier to read

Reply:
well, this is a matter of taste - this reads ok to me, and it doesn't pollute namespace, which I found very useful - no need to pollute code with { ... }